addons/website_customer/views/website_customer_templates.xml

```diff
@@ -196,7 +196,7 @@
                     <label class="cursor-pointer" t-if="industry_dict['industry_id'] and industry_dict['industry_id'][0] == current_industry_id and true or false" t-out="industry_dict['industry_id'][1]"/>
                 </t>
             </a>
-            <div class="dropdown-menu">
+            <div class="dropdown-menu" style="max-height: 150px; overflow-y: auto;">
```

Comment:
Would be cleaner to add it in a separate css file but as this targets stable, I can't modify the manifest of the module. I will do it in master :)